### ISSUE TITLE
PredictionIO install script doesn't work!

### ISSUE THREAD

**Issue by techscientist** [apache/incubator-predictionio#285]
Hi,

The install script used to work before, but now it does not work. In fact, the directories that it creates are automatically deleted by the time the script finishes.

Can you please fix the install script?

**Comment by SimonDeconde** [comment]
Indeed, there are some script errors, eg: 

> ./install.sh: line 241: break: only meaningful in a `for', `while', or `until' loop

Full execution trace here: [pio_install_execution_trace.txt](https://github.com/apache/incubator-predictionio/files/439846/pio_install_execution_trace.txt)

**Comment by rrcook** [comment]
Happened to me as well.

**Comment by gdoslu** [comment]
Download the installation script at 'https://raw.githubusercontent.com/apache/incubator-predictionio/master/bin/install.sh'
Go to the line 281 
Replace https://github.com/PredictionIO/PredictionIO/releases/download/v${PIO_VERSION}/${PIO_FILE}
 with 
http://download.prediction.io/PredictionIO-0.9.6.tar.gz

then save and exit.
Run this commands: 

sudo chmod +x install.sh
./install.sh
source .profile

Installation is finished :)

Check installation by running
pio status

**Comment by EmergentOrder** [comment]
I think you all must be using the install script from master. The script in develop is working, and will install the latest snapshot version (including the jars being put into your local ivy, required for templates to build).

**Comment by dszeto** [comment]
Hi all, I encourage all of you to subscribe and post your questions to user@predictionio.incubator.apache.org for a wider exposure.

For users who do not prefer building from source, please use @gdoslu 's solution. We will be able to make improvements to the latest installer once we have made an Apache release that lives on Apache infrastructure. Thank you for your patience.

**Comment by piotremis** [comment]
There is another problem with installation script...it try to download `https://github.com/PredictionIO/PredictionIO/releases/download/v0.9.6/PredictionIO-0.9.6.tar.gz` file but it doesn't exist...
`echo "Downloading PredictionIO..."
curl -OL https://github.com/PredictionIO/PredictionIO/releases/download/v${PIO_VERSION}/${PIO_FILE}`

**Comment by techscientist** [comment]
@dszeto , do you have any approximate date for the Apache release of PredictionIO? Just wondering...

**Comment by rrcook** [comment]
@gdoslu 's solution did not work, @EmergentOrder 's did.
I changed the url in the quick install, substituted develop for master, it ran.

**Comment by aengle1429** [comment]
Yes, follow @EmergentOrder 's instructions but instead download install.sh from
https://raw.githubusercontent.com/apache/incubator-predictionio/develop/bin/install.sh
